extras/AFC.py

```diff
@@ -584,6 +584,23 @@ def TOOL_LOAD(self, CUR_LANE):
             self.toolhead.manual_move(pos, CUR_EXTRUDER.tool_load_speed)
             self.toolhead.wait_moves()
 
+            # Check if ramming is enabled, if it is go through ram load sequence.
+            # Lane will load until Advance sensor is True
+            # After the tool_stn distance the lane will retract off the sensor to confirm load and reset buffer
+            if CUR_EXTRUDER.tool_start == "buffer":
+                CUR_LANE.extruder_stepper.sync_to_extruder(None)
+                load_checks = 0
+                while CUR_EXTRUDER.tool_start_state == True:
+                    CUR_LANE.move( self.short_move_dis * -1, self.short_moves_speed, self.short_moves_accel )
+                    load_checks += 1
+                    self.reactor.pause(self.reactor.monotonic() + 0.1)
+                    if load_checks > self.tool_max_load_checks:
+                        msg = ''
+                        msg += "Buffer did not become compressed after {} short moves.\n".format(self.tool_max_load_checks)
+                        msg += "Tool may not be loaded"
+                        self.gcode.respond_info("<span class=warning--text>{}</span>".format(msg))
+                        break
+                CUR_LANE.extruder_stepper.sync_to_extruder(CUR_LANE.extruder_name)
```

Comment:
This should probably be outside of the if statement so it syncs back up for either buffer or endstops